mailpoet/assets/js/src/automation/integrations/mailpoet/types/filters.tsx

```diff
@@ -2,6 +2,11 @@
  * The types in this file document the expected return types of specific
  * filters.
  */
+import { State } from '../../../editor/store';
+import { Step } from '../../../editor/components/workflow/types';
+
+// mailpoet.automation.send_email.create_step
+export type SendEmailCreateStepType = (step: Step, state: State) => Step;
```

Comment:
I do not see this type being used in free or premium

Reply:
Oh, yeah, I created it first and then didn't use it, it seems. The issue is I don't assign it to a variable anywhere, anything I could type simply. Do you think I should delete the type or try to use it somehow?